### PR TITLE
Add video to MBRDNA case study

### PR DESCRIPTION
**Pull request by zchase**
As the title states, this PR adds a video to the MBRDNA case study.

### CODE REVIEW

**Comment by cnunciato** [comment]
Since this is a Markdown file, we should use the Hugo shortcode for it.

File: content/case-studies/mercedes-benz.md [hunk 1]
```diff
@@ -29,11 +29,13 @@ sections:
       anchor: developers-in-the-driver-s-seat-programming-the-cloud-with-pulumi
     - label: Conclusion
       anchor: the-road-ahead
-      
+
 aliases:
     - /case-studies/merceded-benz
 ---
 
+{{< youtube "MGA_Bba2GqQ" >}}
```
Comment (justinvp) [inline content/case-studies/mercedes-benz.md:37]: Should this have `?rel=0` per https://github.com/pulumi/docs/blob/master/BLOGGING.md#video ?

```suggestion
{{< youtube "MGA_Bba2GqQ?rel=0" >}}
```

**Comment by cnunciato** [comment]
I just wanted to leave one minor comment in that it feels a teeny bit strange to have a YouTube video embedded in a case study like this, especially up at the top of it like this. Totally not meaning to block, but if I were giving you UX feedback here, I'd suggest maybe dropping using a note to direct users _from_ this page to the YouTube video, or at most, put the video at the bottom of the page with a note explaining what it is and how it relates to the content of the page.

**Comment by cnunciato** [comment]
... or .. not. 😉

**Pull request merged**